boards/phpbb3/bans.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * MyBB 1.8 Merge System
+ * Copyright 2014 MyBB Group, All Rights Reserved
```

Comment:
This copyright year should be 2019:

```suggestion
 * Copyright 2019 MyBB Group, All Rights Reserved
```

Reply:
Updated my files to 2019.

Reply:
Perfect, thanks!